### DISCUSSION TITLE
What display manager and GUI are used in the desktop version

### DISCUSSION THREAD

**Discussion by bhouseski** [Joshua-Riek/ubuntu-rockchip#634] [General]
As the title states, I would like to know which desktop manager and GUI are used in the Ubuntu desktop image?  I am trying to recreate this from the server image, as I am unable to get video out on my Radxa Zero 3W.

**Comment by Xeue** [comment]
Wayland and Gnome. The ubuntu defaults. You can pretty much just do "sudo apt install ubuntu-desktop"
